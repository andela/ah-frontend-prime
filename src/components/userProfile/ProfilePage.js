import React, { Component } from "react";

import ProfileContainer from "../../containers/profile/ProfileContainer";

export class ProfilePage extends Component {
  render() {
    return (
      <div>
        <ProfileContainer />
      </div>
    );
  }
}
export default ProfilePage;
