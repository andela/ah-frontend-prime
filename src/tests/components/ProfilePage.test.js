import { shallow } from "enzyme";
import React from "react";

import ProfilePage from "../../components/userProfile/ProfilePage";

describe("profile page tests", () => {
  const props = {
    onSubmit: jest.fn(),
    errors: null,
    generalError: ""
  };
  it("should render without crushing", () => {
    const wrapper = shallow(<ProfilePage {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
