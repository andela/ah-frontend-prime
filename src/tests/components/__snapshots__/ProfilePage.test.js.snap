// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile page tests should render without crushing 1`] = `ShallowWrapper {}`;
